pkg/systemlogmonitor/logwatchers/journald/log_watcher.go

```diff
@@ -135,27 +135,28 @@ func (j *journaldWatcher) watchLoop() {
 }
 
 const (
-	// defaultJournalLogPath is the default path of journal log.
-	defaultJournalLogPath = "/var/log/journal"
-
 	// configSourceKey is the key of source configuration in the plugin configuration.
 	configSourceKey = "source"
 )
 
 // getJournal returns a journal client.
 func getJournal(cfg types.WatcherConfig, startTime time.Time) (*sdjournal.Journal, error) {
 	// Get journal log path.
-	path := defaultJournalLogPath
-	if cfg.LogPath != "" {
```

Comment:
I think the idea here is that if someone doesn't want to use `/var/log/journal`, he can just set `cfg.LogPath`. Can you explain more on your use case and why current approach is not enough?

Reply:
Some of our nodes have different log paths from others. In both cases, they use the log path that is returned by NewJournal().

It would seem better to me to fall back to journald's configured path if path is not specified, rather than hard-code a default.

Reply:
That's fair.